aten/src/ATen/native/quantized/cpu/qnormalization.cpp

```diff
@@ -55,13 +55,17 @@ Tensor quantized_group_norm_impl(
     double output_scale,
     int64_t output_zero_point) {
 
-  const auto input_ndim = qx.dim();
+  const auto& qx_contig = qx.is_contiguous() ? qx : qx.contiguous();
+  const auto& weight_contig = weight.is_contiguous() ? weight : weight.contiguous();
+  const auto& bias_contig = bias.is_contiguous() ? bias : bias.contiguous();
```

Comment:
this is probably the same as just always calling contiguous: https://codebrowser.bddppq.com/pytorch/pytorch/aten/src/ATen/native/TensorProperties.cpp.html#83

Reply:
this comment is still valid